rune/libenclave/internal/runtime/pal/skeleton/sgxutils.c

```diff
@@ -75,6 +77,41 @@ void get_sgx_xfrm_by_cpuid(uint64_t *xfrm)
 	*xfrm = try_get_xcr0();
 }
 
+uint32_t sgx_calc_ssaframesize(uint32_t miscselect, uint64_t xfrm)
+{
+	uint32_t sgx_xsave_size_tbl[63];
+	uint32_t size_max = PAGE_SIZE;
+	int cpu_info[4] = {0, 0, 0, 0};
+	uint32_t size;
+	int i;
+
+	for (i = 2; i < 63; i++) {
+		__cpuidex(cpu_info, 0x0D, i);
+		if ((1 << i) & xfrm)
+			sgx_xsave_size_tbl[i] = cpu_info[0] + cpu_info[1];
```

Comment:
Using sgx_xsave_size_tbl[] is meaningless. Why not just use a variable called xsave_offset to save the result?